test/functional/fixtures/page-error/testcafe-fixtures/page-error-test.js

```diff
@@ -0,0 +1,23 @@
+import { ClientFunction } from 'testcafe';
+import { expect } from 'chai';
+
+fixture `Page error`
+    .page `http://localhost:3000/page-error/pages/index.html`;
+
+test('Do not handle', async t => {
+    await t.click('#unreachable-page-link');
+});
+
+// TODO: action error recovery (GH-567)
+test('Handle', async t => {
+    try {
+        await t.click('#unreachable-page-link');
+    }
+    catch (err) {
+        await t.navigateTo('handled.html');
```

Comment:
This is scenario when we handle pageError and resume the test from the `about:error` page. But we this scenario will work when we fix #567. Do we need have this test here now?